cli/ops/fs.rs

```diff
@@ -684,20 +710,28 @@ fn op_symlink(
   _zero_copy: Option<ZeroCopyBuf>,
 ) -> Result<JsonOp, OpError> {
   let args: SymlinkArgs = serde_json::from_value(args)?;
-  let oldname = deno_fs::resolve_from_cwd(Path::new(&args.oldname))?;
-  let newname = deno_fs::resolve_from_cwd(Path::new(&args.newname))?;
+  let oldpath = resolve_from_cwd(Path::new(&args.oldpath))?;
+  let newpath = resolve_from_cwd(Path::new(&args.newpath))?;
+
+  state.check_write(&newpath)?;
 
-  state.check_write(&newname)?;
-  // TODO Use type for Windows.
-  if cfg!(not(unix)) {
-    return Err(OpError::other("Not implemented".to_string()));
-  }
   let is_sync = args.promise_id.is_none();
   blocking_json(is_sync, move || {
-    debug!("op_symlink {} {}", oldname.display(), newname.display());
+    debug!("op_symlink {} {}", oldpath.display(), newpath.display());
     #[cfg(unix)]
-    std::os::unix::fs::symlink(&oldname, &newname)?;
-    Ok(json!({}))
+    {
+      use std::os::unix::fs::symlink;
+      symlink(&oldpath, &newpath)?;
+      Ok(json!({}))
+    }
+    // TODO Implement symlink, use type for Windows
+    #[cfg(not(unix))]
+    {
+      // Unlike with chmod/chown, here we don't
+      // require `oldpath` to exist on Windows
+      let _ = oldpath; // avoid unused warning
+      return Err(OpError::other("Not implemented".to_string()));
```

Comment:
use `OpError::not_implemented`

Reply:
See the remark about this in the PR header. That triggers that some tests on symlink have to check for "not implemented" messages and others have to check for "Not implemented" messages. Fixing that requires fixing capitalization of other "Not implemented" checks across the codebase. I had a commit that did this in the previous refactoring PR, but now at your prompting I've rebased it out and will submit it separately.

Reply:
Ah - I see now! Cool.